### PR TITLE
Implementation Get Operation to Obtain An User With Given Id

### PR DESCRIPTION
**Pull request by blancaagc**
I added a functionality to obtain an user with an ID and return an error when the ID is wrong.

### CODE REVIEW

**Review by luisgalindoluna — APPROVED** [review]
Looks good to me

File: src/main/java/com/agilethought/atsceapi/model/User.java [hunk 1]
```diff
@@ -3,12 +3,15 @@
 import lombok.Data;
 import org.springframework.data.annotation.Id;
 import org.springframework.data.mongodb.core.mapping.Document;
+import javax.persistence.GeneratedValue;
```
Comment (ivanflores15) [inline src/main/java/com/agilethought/atsceapi/model/User.java:6]: your project is broken, please fix it. 
image

**Review by ivanflores15 — CHANGES_REQUESTED** [review]

**Review by ivanflores15 — CHANGES_REQUESTED** [review]
your project is broken, please fix it. 
image

**Comment by ivanflores15** [comment]
Actually, your are  one commit behind delevop. To solve, this issue... 

- pull all to your delelop local branch
- merge your feature branch into develop local branch
- Solve conflicts
- Commit and push your changes

image

**Review by ivanflores15 — CHANGES_REQUESTED** [review]
image

you removed your method to find users by Id. In that way your code doesn't do what is supposed

**Review by ivanflores15 — APPROVED** [review]
looks good for me

**Review by luisgalindoluna — APPROVED** [review]
Looks fine, good work.

**Review by EmiliaGuillen-dev — APPROVED** [review]
looks good 👍

**Pull request merged**